features/steps/component.go

```diff
@@ -61,67 +70,74 @@ func (c *Component) initService(ctx context.Context) error {
 		return nil
 	}
 
-	log.Info(ctx, "config read", log.Data{"cfg": cfg})
-
+	cfg.KafkaConfig.Addr = []string{"kafka:9092"}
```

Comment:
Could these not be passed as environment variables?

Reply:
Yes, I will do the change